specs/src/pages/widgets/frequently-bought-together.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: ../../layouts/WidgetLayout.astro
+title: FrequentlyBoughtTogether
+type: widget
+html: |
+  <div class="ais-FrequentlyBoughtTogether">
+    <h3 class="ais-FrequentlyBoughtTogether-title">Frequently bought together</h3>
+    <div class="ais-FrequentlyBoughtTogether-container">
+      <ol class="ais-FrequentlyBoughtTogether-list">
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 5477500: Amazon - Fire TV Stick with Alexa Voice Remote - Black
+        </li>
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 4397400: Google - Chromecast - Black
+        </li>
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 4397400: Google - Chromecast - Black
+        </li>
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 5477500: Amazon - Fire TV Stick with Alexa Voice Remote - Black
+        </li>
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 4397400: Google - Chromecast - Black
+        </li>
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 4397400: Google - Chromecast - Black
+        </li>
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 5477500: Amazon - Fire TV Stick with Alexa Voice Remote - Black
+        </li>
+        <li class="ais-FrequentlyBoughtTogether-item">
+          Hit 4397400: Google - Chromecast - Black
+        </li>
+      </ol>
+    </div>
+  </div>
+classes:
+  - name: .ais-FrequentlyBoughtTogether
+    description: the root div of the widget
+  - name: .ais-FrequentlyBoughtTogether-title
+    description: the title heading of the widget
+  - name: .ais-FrequentlyBoughtTogether-container
+    description: the container for the list of results
+  - name: .ais-FrequentlyBoughtTogether-list
+    description: the list of results
+  - name: .ais-FrequentlyBoughtTogether-item
+    description: the hit list item
+options:
```

Comment:
We're missing multiple options that can be passed to FBT, don't we?

Reply:
I've added props and translations that exist right now. I've omitted header/item/view components as they are templates and not documented in other widgets in the specs.